### ISSUE TITLE
[Bug]: NPC animations not working in shopping district

### ISSUE THREAD

**Issue by Griffisaurus** [XRFoundation/XREngine#6306]
NPC avatar is broken in the shopping district. The NPCs in the theater still work but I don't think we have changed the animation on those recently.  Just now the shopping district had the animation to walk left, which is different than the original wave.  Still, I changed back to wave and the NPC remained in T pose.

![image](https://user-images.githubusercontent.com/92340542/171741404-43babc0f-eb34-423c-b98a-e7619941f8d8.png)

**Comment by NPatel10** [comment]
@Griffisaurus, I followed below steps to recreate this issue but can't do it,
1. Entered into below location and find that the NPC is running `dance2` animation in loop.
    https://dev.theoverlay.io/location/shopping-center
2. I opened this location in editor (https://dev.theoverlay.io/editor/shopping-center/shopping-district-public) and tried to set loop animation to `wave`. then saved it.
3. I re entered into above location and find the NPC is running `wave` animation in loop.

Provide me some more steps to reproduce it.
PS - I am reverting my change and setting the animation to `dance2`.
